extension/src/background/index.ts

```diff
@@ -50,3 +53,11 @@ export const initInstalledListener = () => {
   });
   browser?.runtime?.onInstalled.addListener(migrateFriendBotUrlNetworkDetails);
 };
+
+export const initInitAlarmListener = (store: Store) => {
```

Comment:
probably want to rename this to `initAlarmListener`

Reply:
😄 thanks